src/components/Navigation/NavigationHeadline.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
why is this a standalone component? we use a `title` or `label`-prop for header's in similiar components.